scripts/gitops-secrets-gen.sh

```diff
@@ -111,4 +111,9 @@ if [ ! -f "${UC_DEPLOY}/secrets/${DEPLOY_NAME}/secret-metallb.yaml" ]; then
     echo "---" > "${UC_DEPLOY}/secrets/${DEPLOY_NAME}/secret-metallb.yaml"
 fi
 
+if [ ! -f "${UC_DEPLOY}/secrets/${DEPLOY_NAME}/secret-nautobot-env.yaml" ]; then
+    echo "Creating nautobot-env secret placeholder"
+    echo "---" > "${UC_DEPLOY}/secrets/${DEPLOY_NAME}/secret-nautobot-env.yaml"
```

Comment:
You'll want to create a secret with no data fields rather than an empty file. The empty file is good for a Helm values.yaml override but not for a secret that's being iterated.